### ISSUE TITLE
mod not working

### ISSUE THREAD

**Issue by seclorumis** [FlansMods/FlansMod#1267]
the mod is not working, the textures dont work, i put everything in the folders i downloaded into the mod folder, and nothing worked. please tell me how to fix it.

**Comment by JValor2417** [comment]
Hi @seclorumis,

I have a possible solution. 
The way Flans mod loads and creates content in game is by using content packs from the “ Flan “ folder in your root directory. 
Try typing %AppData% in your devices internal search bar ( it should work on windows and Mac for sure, ) and it should bring you to a series of folders where your MC launcher is in. For example, using technic launcher will most likely result in your launcher items being in “ .technic “ and launching the game from the vanilla MC launcher with forge installed will most likely be labeled “ .minecraft “.  Click the folder with a name relative to your launcher, and find the “ Flan “ folder as mentioned above. All of your content packs ( ModernWarfare pack, WWII pack, etc ) go here as they contain contents for the mod ( guns, vehicles, ammo, sounds, textures etc. ) now, double check that you have FlansMod.JAR in your mods folder. 

TLDR: All content packs go into your “ Flan “ folder that’s generated when loading MC with the mod, and FlansMod.JAR goes into the “ mods “ folder.

**Comment by ChrisLane** [comment]
Please follow the issue reporting guidelines to help us to provide support.
https://github.com/FlansMods/FlansMod/wiki/Issue-Reporting
